### ISSUE TITLE
Attribute error in limit

### ISSUE THREAD

**Issue by proy87** [sympy/sympy#19586]
`limit(x**(2**x*3**(-x)),x,oo)` raises `AttributeError: 'Pow' object has no attribute 'ceiling'`. Tested using Sympy 1.6
Everything was OK in 1.5 version.

**Comment by sachin-4099** [comment]
This is not happening on `1.7.dev`:
```
In [1]: limit(x**(2**x*3**(-x)),x,oo)
Out[1]: 1
```

**Comment by sachin-4099** [comment]
This diff fixes the issue:
```
--- a/sympy/functions/elementary/exponential.py
+++ b/sympy/functions/elementary/exponential.py
@@ -431,7 +431,7 @@ def _eval_is_extended_positive(self):
     def _eval_nseries(self, x, n, logx):
         # NOTE Please see the comment at the beginning of this file, labelled
         #      IMPORTANT.
-        from sympy import limit, oo, Order, powsimp, Wild, expand_complex
+        from sympy import ceiling, limit, oo, Order, powsimp, Wild, expand_complex
         arg = self.args[0]
         arg_series = arg._eval_nseries(x, n=n, logx=logx)
         if arg_series.is_Order:
@@ -446,7 +446,7 @@ def _eval_nseries(self, x, n, logx):
         except NotImplementedError:
             cf = 0
         if cf and cf > 0:
-            nterms = (n/cf).ceiling()
+            nterms = ceiling(n/cf)
         exp_series = exp(t)._taylor(t, nterms)
         r = exp(arg0)*exp_series.subs(t, arg_series - arg0)
         if cf and cf > 1:
```

**Comment by oscarbenjamin** [comment]
@sachin-4099 can you open a pull request for the 1.6 branch that will fix this?

**Comment by oscarbenjamin** [comment]
This is now fixed on the 1.6 branch for 1.6.1 by #19604. I'm leaving this issue open until a test for this case is added on master.
